src/App.js

```diff
@@ -1,12 +1,24 @@
+import About from "./pages/About";
+import EditProfile from "./pages/EditProfile";
+import Forum from "./pages/Forum";
+import Home from "./pages/Home";
 import Login from "./pages/Login";
-import Test from "./pages/Test";
+import Users from "./pages/Users";
+import UsersProfile from "./pages/UsersProfile";
+import Profile from "./pages/Profile";
 import { Route, Routes } from "react-router-dom";
 
 function App() {
   return (
     <Routes>
-      <Route path="/" element={<Login />} />
-      <Route path="/test" element={<Test />} />
+      <Route path="/" element={<Home />} />
+      <Route path="/about" element={<About />} />
+      <Route path="/editprofile" element={<EditProfile />} />
+      <Route path="/forum" element={<Forum />} />
+      <Route path="/profile" element={<Profile />} />
+      <Route path="/users" element={<Users />} />
+      <Route path="/users/:dir" element={<UsersProfile />}></Route>
```

Comment:
Can rewrite line 20 to
`<Route path="/users/:dir" element={<UsersProfile />} />`